Makefile

```diff
@@ -0,0 +1,49 @@
+KUEUE_REPO=https://github.com/kubernetes-sigs/kueue.git
+KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
+
+KUBECTL_VERSION := $(shell curl -L -s https://dl.k8s.io/release/stable.txt)
+OS := $(shell dpkg --print-architecture)
+
+ifeq ($(OS),amd64)
+	KUBECTL_URL = "https://dl.k8s.io/release/$(KUBECTL_VERSION)/bin/linux/amd64/kubectl"
+endif
+ifeq ($(OS),arm)
+	KUBECTL_URL = "https://dl.k8s.io/release/$(KUBECTL_VERSION)/bin/linux/arm64/kubectl"
+endif	
+
+PROJECT_DIR := $(realpath $(shell dirname $(firstword $(MAKEFILE_LIST))))
+
+BIN_PATH=$(PROJECT_DIR)/bin
+
+.PHONY: install install_kjob install_kueuectl install_gcloud check_python update-path
+
+install: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install
+
+pip-install:
+	pip install .
+
+install-kjob: install-kubectl
+	git clone $(KUEUE_REPO) $(KUEUE_TMP_PATH)
+	make -C $(KUEUE_TMP_PATH)/cmd/experimental/kjobctl kubectl-kjob
+	mv $(KUEUE_TMP_PATH)/cmd/experimental/kjobctl/bin/kubectl-kjob $(BIN_PATH)/kubectl-kjob
+	rm -rf $(KUEUE_TMP_PATH)
+
+mkdir-bin:
+	test -d $(BIN_PATH) || mkdir $(BIN_PATH)
+
+install-kubectl:
+	curl -LO $(KUBECTL_URL)
+	chmod +x kubectl
+	mv ./kubectl $(BIN_PATH)/kubectl
+
+install-kueuectl: install-kubectl
+	git clone $(KUEUE_REPO) $(KUEUE_TMP_PATH)
+	make -C $(KUEUE_TMP_PATH) kueuectl
+	mv $(KUEUE_TMP_PATH)/bin/kubectl-kueue $(BIN_PATH)/kubectl-kueue
+	rm -rf $(KUEUE_TMP_PATH)
```

Comment:
I would propose to use release version https://kueue.sigs.k8s.io/docs/reference/kubectl-kueue/installation/#installing-from-release-binaries.